src/main/java/jenkins/plugins/office365connector/utils/Parser.java

```diff
@@ -1,29 +1,22 @@
 package jenkins.plugins.office365connector.utils;
 
-import hudson.model.AbstractBuild;
 import hudson.model.Cause;
-import jenkins.plugins.office365connector.model.Fact;
-
-import java.io.File;
 import java.util.List;
 
-/**
- * Created by olegfeferman on 23.11.2017.
- */
 public class Parser {
 
     public String getAuthor(List<Cause> causesList) {
         StringBuilder allCauses = new StringBuilder();
-        for (int i = 0; i < causesList.size(); i++) {
-            allCauses.append(causesList.get(i).getShortDescription());
+        for( Cause cause : causesList){
```

Comment:
for (Cause cause : causeList) {

Reply:
Thanks! Corrected this specific case. IDEA is not free, but I found how to do this in netbeans :)

Reply:
The community edition is: `Community Edition is open-source, licensed under Apache 2.0. Projects like Android and Swift use Apache 2.0, so you're in good company. It can also be used for commercial development.`
![image](https://user-images.githubusercontent.com/1661688/33481394-8c5241ea-d694-11e7-80b3-b77e7c1e6192.png)

Reply:
Thanks